mininext/topo.py

```diff
@@ -17,15 +17,16 @@ def __init__(self, nopts=None, **opts):
         BaseTopo.__init__(self, **opts)
 
     # Override addHost so that constructor defaults to MiniNExT host
-    def addHost(self, name, cls=Host, **opts):
+    def addHost(self, name, **opts):
         """Adds a host using the MiniNExT host constructor.
            name: host name
            cls: host constructor
            opts: host options
            returns: host name"""
         if not opts and self.hopts:
             opts = self.hopts
-        return BaseTopo.addNode(self, name, cls=cls, **opts)
+        opts['cls'] = Host
```

Comment:
Good point! I like checking for cls instead, because this function is meant to override another, so I'd prefer to have the signatures of the methods matching to avoid confusion. Not adamant about it though.